### ISSUE TITLE
Pre-set tracks are appeared twice

### ISSUE THREAD

**Issue by telk5093** [OpenRCT2/OpenRCT2#14446]
**OS:** Windows 10
**Version:** 0.3.3
**Commit/Build:** 9b2c946

![image](https://user-images.githubusercontent.com/10726524/113753550-30ad4280-9749-11eb-84f5-005c5a83e8cb.png)

This issue is not came from me, two of korean users have reported this exactly same issue.
Second report post was at my website: https://telk.kr/tb/free/2623

He said, preset tracks are appeared twice even though they have the exactly same file when he checked with debug tool enabled.
He said their file path were the same and so were the extension(TD6)

But it is not reproducible for me. I think maybe neither you would be.
I am totally have no idea, but anyway I file this issue here.

I'll translate what he said at [this comment](https://telk.kr/tb/free/2623#tb_comment-26619):
> What I did were
> First, I installed RCT2: TTP that is already I have in Steam,
> then downloaded ZIP archive of 9b2c946 via TELKLAND site (Note: it is exactly the same url with openrct2.org),
> then extracted it onto install path of RCT2: TTP of Steam (Note: ``~~/steamapp/common/RollerCoaster Tycoon 2: Triple Thrill Pack``),
> then downloaded [RCT1 Scenarios converted for RCT2](https://telk.kr/download/orct/aHR0cCUzQSUyRiUyRnd3dy5tZWRpYWZpcmUuY29tJTJGZG93bmxvYWQlMkYzY3c0N3NkMnd0M2h1ZXMlMkZFeGFjdFJDVDFSZWNyZWF0aW9ucy56aXA=)
> and then overwrote them into Scenarios folder in RCT2's install directory of Steam.
> (I don't have RCT1 and did not set the path on OpenRCT2)
> After these steps, two same preset tracks are appeared

**Comment by RenanLazarotto** [comment]
Same happens for me, default track shows twice even though I only have one file for them. I copied all tracks from RCT1 and 2 into OpenRCT2 folder, deleted the ones with the same name but different versions (TD4 and TD6, kept the TD6) but the problem persisted. I also tried to delete tracks.idx from C:\Users\(username)\Documents\OpenRCT2 but it didn't help.

**Comment by telk5093** [comment]
Reported this issue today again, left a reference link (Korean): https://gall.dcinside.com/mgallery/board/view/?id=rct&no=27078

**Comment by Gymnasiast** [comment]
Looks like this was the result of mixing RCT2 and OpenRCT2 files - which we recommend against.

**Comment by wsjjoy87** [comment]
> Looks like this was the result of mixing RCT2 and OpenRCT2 files - which we recommend against.

Yes. Today I tried to install rct2 from steam by many ways with OpenRCT2. I found that original tracks duplicated to use OpenRCT2 in steam like following steam start option.

I think it seems like OpenRCT2 tracks referenced "both" installed and OpenRCT2 folder and working like not "OR" referecened "BOTH". So, if I move all OpenRCT2 files in steam library folder, when OpenRCT2 started, first referenced to OpenRCT2 folder : steam library and second referenced installed rct2 : steam library. 
So, it become 2 same track. 
I think if you follow relating steam, it will be happen. In my case, always happen.
This case maybe rct2.exe referenced track and OpenRCT2 also referenced installed folder. 

I think it also could be considered bug of exceptional environment.

**Comment by lucamicino** [comment]
> Looks like this was the result of mixing RCT2 and OpenRCT2 files - which we recommend against.

I am having this issue as well. Originally I pasted OpenRCT2 files into the RCT2 folder in my Steam directory, which was causing preset tracks to show up twice, then I reinstalled the game as well as OpenRCT2, but instead I put OpenRCT2 in my documents folder rather than having it in the game folder, and used a launch argument from Steam to run OpenRCT2 but still having this issue. What am I doing wrong?
